charters/wot-ig-2019.html

```diff
@@ -167,278 +179,202 @@ <h1 id="title">Proposed Web of Things Interest Group Charter</h1>
 
+          <p>To succeed in establishing inter-platform standards, W3C needs to coordinate with IoT alliances and standards development organizations. The following provides examples of organizations with which the Interest Group has already had significant coordination, during the period of the first charter. A <a href="https://www.w3.org/WoT/IG/wiki/How_does_W3C_compare_to_other_organizations">longer list</a> is available on the Interest Group wiki.</p>
 
-        </div>
--->
+          <dl>
+            <dt><a href="https://datatracker.ietf.org/doc/charter-irtf-t2trg/">IRTF Thing to Thing Research Group</a></dt>
+            <dd>For coordination of matters of mutual interest in relation to the Web of Things.</dd>
+            <dt><a href="http://openconnectivity.org">Open Connectivity Foundation</a></dt>
+            <dd>For collaboration on integrating OCF based platforms within the Web of Things, including platform metadata and approaches for enabling semantic interoperability, and end to end security across platforms.</dd>
+            <dt><a href="https://echonet.jp/english/">ECHONET Consortium</a></dt>
+            <dd>For collaboration on integrating ECHONET Consortium based platforms within the Web of Things, including platform metadata and approaches for enabling semantic interoperability, and end to end security across platforms.</dd>
+            <dt><a href="http://www.iiconsortium.org">Industrial Internet Consortium</a></dt>
+            <dd>For collaboration on a testbed for the Web of Things.</dd>
+            <dt><a href="https://opcfoundation.org">OPC Foundation</a></dt>
+            <dd>For collaboration on semantic interoperability and end to end security across platforms.</dd>
+            <dt><a href="http://www.plattform-i40.de/I40/Navigation/EN/Home/home.html">Plattform Industrie 4.0</a></dt>
+            <dd>For collaboration on use cases and requirements for smart manufacturing, including semantic interoperability and end to end security across platforms.</dd>
+            <dt><a href="http://www.onem2m.org">oneM2M</a></dt>
+            <dd>For collaboration on integrating M2M based platforms within the Web of Things, including platform metadata and approaches for enabling semantic interoperability, and end to end security across platforms.</dd>
+            <dt><a href="https://www.etsi.org">ETSI</a></dt>
+            <dd>For collaboration on IoT, M2M, security, and semantic interoperability.</dd>
+            <dt><a href="https://www.omaspecworks.org">OMA SpecWorks</a></dt>
+            <dd>For collaboration on integrating LWM2M based platforms with the Web of Things.</dd>
+            <dt><a href="http://www.opengeospatial.org/">Open Geospatial Consortium</a></dt>
+            <dd>For coordination on the OGC SensorThings API.</dd>
+            <dt><a href="http://www.gsma.com">GSMA</a></dt>
+            <dd>For coordination in respect to the interests of Network Operators.</dd>
+          </dl>
+        </section>
```

Comment:
Part of me wants to add a few more, like EdgeX and OpenAPI, but as long as this is meant to be an open-ended list, this is good enough.